utils/user-agent/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { IncomingMessage } from 'http';
+import crawlers from 'crawler-user-agents';
+
+const checkUserAgentIsABot = (req: IncomingMessage) => {
+  const userAgent = req.headers['user-agent'];
+  if (!userAgent) {
+    return false;
+  }
+  for (let i = 0; i < crawlers.length; i++) {
```

Comment:
J'utiliserais la méthode [Array.some](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Array/some) pour faire quelque chose de plus lisible que cette boucle for

Reply:
Bien vu, merci